pkg/services/tag/tagimpl/xorm_store.go

```diff
@@ -0,0 +1,35 @@
+package tagimpl
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/sqlstore/db"
+	"github.com/grafana/grafana/pkg/services/tag"
+)
+
+type sqlStore struct {
+	db db.DB
+}
+
+func (s *sqlStore) EnsureTagsExist(ctx context.Context, tags []*tag.Tag) ([]*tag.Tag, error) {
+	err := s.db.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		for _, tagelement := range tags {
```

Comment:
nit, I would suggest using camelCase to become more readable
```suggestion
		for _, tagElement := range tags {
```